src/pudl/output/eia860.py

```diff
@@ -368,13 +367,15 @@ def generators_eia860(
 
 
 def fill_generator_technology_description(gens_df: pd.DataFrame) -> pd.DataFrame:
-    """Fill in missing ``technology_description`` based on generator and energy source.
+    """Fill in missing ``technology_description`` based by backfilling & unquie mapping.
 
     Prior to 2014, the EIA 860 did not report ``technology_description``. This
     function backfills those early years within groups defined by ``plant_id_eia``,
-    ``generator_id`` and ``energy_source_code_1``. Some remaining missing values are
-    then filled in using the consistent, unique mappings that are observed between
-    ``energy_source_code_1`` and ``technology_type`` across all years and generators.
+    ``generator_id`` and ``energy_source_code_1``.
+
+    Some remaining missing values are then filled in using the consistent,
+    unique mappings that are observed between ``energy_source_code_1``,
+    ``prime_mover_code`` and ``technology_type`` across all years and generators.
 
     As a result, more than 95% of all generator records end up having a
     ``technology_description`` associated with them.
```

Comment:
Do you know what the coverage is now that you've integrated `prime_mover_code`?

Reply:
ha its 97%... which is a HUGE improvement from 96%.

Reply:
with moving the PM code....  this is now 98.1% 😎